### DISCUSSION TITLE
How to control notifications for cron triggered workflows?

### DISCUSSION THREAD

**Discussion by pcstout** [community/community#25351] [Actions]
<p>How can I specify a list of users that get notified when a cron triggered workflow fails?<br>
Right now it appears that whichever user last pushed the codebase is the one and only user that gets notified.</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/pcstout">@pcstout</a>,</p>
<p>This is by designed. As a workaround, you can add a step to send out the notification to a list of users.<br>
You can find a lot of ‘send email’ actions in the <a href="https://github.com/marketplace?query=email&amp;type=actions" rel="nofollow noopener">marketpace</a>.<br>
For example, you can use:</p>
<pre><code class="lang-auto">      - name: failed step
        run: |
          exit 1
      - name: Send mail
        if: failure()
        uses: dawidd6/action-send-mail@v2
        with:
          server_address: smtp.gmail.com
          server_port: 465
          username: ${{secrets.MAIL_USERNAME}}
          password: ${{secrets.MAIL_PASSWORD}}
          subject: Github Actions job result
          # Literal body:
          body: Build job of ${{github.repository}} Failed!
          # Read file contents as body:
          # body: file://README.md
          to: wexxxxx@outlook.com,zxdxxxx5@outlook.com
          from: Github action # &lt;user@example.com&gt;
          # Optional content type (defaults to text/plain):
          # content_type: text/html
          # Optional attachments:
          # attachments: attachments.zip,git.diff,./dist/static/main.js
</code></pre>
<p>My workflow for your reference: <a href="https://github.com/weide-zhou/ticket13/runs/764453779?check_suite_focus=true#step:3:29" rel="nofollow noopener">https://github.com/weide-zhou/ticket13/runs/764453779?check_suite_focus=true#step:3:29</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/b5319f182e3a4a7a9a0e2144fc0d16d98ff603fe.png" data-download-href="/uploads/short-url/pQUJBOLgH1h5xZp0ojLjO5H5Xlk.png?dl=1" title="Sendemail" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181090574-8f6f8978-304c-4c0b-af03-75b9652de0fa.png" alt="Sendemail" data-base62-sha1="pQUJBOLgH1h5xZp0ojLjO5H5Xlk" width="690" height="146" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/b/b5319f182e3a4a7a9a0e2144fc0d16d98ff603fe_2_690x146.png, https://aws1.discourse-cdn.com/github/original/2X/b/b5319f182e3a4a7a9a0e2144fc0d16d98ff603fe.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/b/b5319f182e3a4a7a9a0e2144fc0d16d98ff603fe.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/b5319f182e3a4a7a9a0e2144fc0d16d98ff603fe_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Sendemail</span><span class="informations">978×208 28.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Thanks</p>

**Comment by pcstout** [comment]
<p>Thank <a class="mention" href="/weide-zhou">@weide-zhou</a>! This is exactly what I needed. Works perfectly.</p>

**Comment by dhimmel** [comment]
> Right now it appears that whichever user last pushed the codebase is the one and only user that gets notified

The [Notifications for workflow runs](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/notifications-for-workflow-runs) docs provide a bit more clarity on how scheduled workflow notifications operate:

> Notifications for scheduled workflows are sent to the user who initially created the workflow. If a different user updates the cron syntax in the workflow file, subsequent notifications will be sent to that user instead. If a scheduled workflow is disabled and then re-enabled, notifications will be sent to the user who re-enabled the workflow rather than the user who last modified the cron syntax.

I was able to disable and then enable a scheduled workflow to switch notifications to go to my account:

> ![image](https://user-images.githubusercontent.com/1117703/186795194-d3fb7041-bb42-4dcc-8a99-74225248dcb7.png)

>![image](https://user-images.githubusercontent.com/1117703/186795246-931f2563-fa3e-45e5-8a21-2611e762ab97.png)
